core/src/main/java/org/elasticsearch/action/search/SearchActionListener.java

```diff
@@ -27,8 +27,9 @@
  * received by this listener.
  */
 abstract class SearchActionListener<T extends SearchPhaseResult> implements ActionListener<T> {
+
     private final int requestIndex;
-    private final SearchShardTarget searchShardTarget;
+    protected final SearchShardTarget searchShardTarget;
```

Comment:
given my comment below I think this is unneeded and can be private without a getter